impeller/renderer/backend/gles/sampler_library_gles.cc

```diff
@@ -17,6 +19,14 @@ SamplerLibraryGLES::~SamplerLibraryGLES() = default;
 // |SamplerLibrary|
 std::shared_ptr<const Sampler> SamplerLibraryGLES::GetSampler(
     SamplerDescriptor descriptor) {
+  if (descriptor.width_address_mode == SamplerAddressMode::kDecal ||
+      descriptor.height_address_mode == SamplerAddressMode::kDecal ||
+      descriptor.depth_address_mode == SamplerAddressMode::kDecal) {
+    VALIDATION_LOG << "SamplerAddressMode::kDecal is not supported by the "
```

Comment:
A TODO linking to a GItHub issue would probably be helpful here.

Reply:
Done & filed this issue: https://github.com/flutter/flutter/issues/129358